src/lib/components/Carousel/imageCarousel.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+	import type { ImageMetadata } from '$lib/types/ImageCarousel';
+
+	export let images: ImageMetadata[] = [];
+  export let carouselId: string;
+</script>
+
+<div id={carouselId} class="carousel slide carousel-fade relative pb-4" data-bs-ride="carousel">
+	<!-- create slideshow at the bottom of the carousel users can click through -->
+	<div
+		class="carousel-indicators absolute right-0 bottom-0 left-0 flex justify-center p-0 mb-4 pb-4"
+	>
+		{#each images as _, i}
```

Comment:
Sveltekit way of [looping through each element](https://svelte.dev/tutorial/each-blocks) of an array. Because we're generating buttons here we don't need any metadata from `images` - we just want to generate `i` number of buttons.

As in vanilla JS array functions (`.map()`, `.filter()`, etc), the 1st parameter is the data, the 2nd parameter is the number of loops